app/controllers/summaries_controller.rb

```diff
@@ -0,0 +1,56 @@
+class SummariesController < ApplicationController
+  def new
+    @summary = Summary.new
+    authorize @summary
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @summary = Summary.new(summary_params)
+    @post.summary = @summary
```

Comment:
This is good. I think you can clean up the code a bit by combining both of these statements though:

```ruby
@summary = @post.build_summary(summary_params)
```